content/en/docs/tasks/access-application-cluster/communicate-containers-same-pod-shared-volume.md

```diff
@@ -7,7 +7,7 @@ weight: 110
 {{% capture overview %}}
 
 This page shows how to use a Volume to communicate between two Containers running
-in the same Pod.
+in the same Pod. See also how to allow processes to communicate by [sharing process namespace](/docs/tasks/configure-pod-container/share-process-namespace/) between containers.
```

Comment:
This extra text seems like it's a better fit for the `whatsnext` section than for the page overview.
How about keeping just the `whatsnext` changes?